core/templates/pages/skill-editor-page/skill-preview-tab/skill-preview-tab.component.ts

```diff
@@ -52,6 +51,7 @@ angular.module('oppia').component('skillPreviewTab', {
         QuestionObjectFactory, QuestionPlayerEngineService,
         SkillEditorStateService, StateCardObjectFactory, UrlService) {
       var ctrl = this;
+      var questionCount = 20;
```

Comment:
I think constants should be in all caps.

Reply:
Done